### DISCUSSION TITLE
migrate media

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#959] [Q&A]
Migration scenario: How to migrate media (including image type and svg type) in the previous CMS system to payload? I know it can be done using the local payload api,
```
payload.create({
    collection: 'media',
    data: {
      alt: fileName,
      slug: slug,
    },
    filePath, // full, absolute file path.
  });
```
But this method is more cumbersome, I need to initialize the payload first.

Currently, the migration is done by nodejs + mongodb. In this way, how to migrate the midia type?

**Comment by denolfe** [comment]
Hey @zhounewz , if you're not wanting to initialize payload first in order to use the local API, you could use `fetch` to perform a POST call against your media collection. Is this an option for you?
